prisma/seed.ts

```diff
@@ -23,12 +23,28 @@ async function seed() {
     data: [
       {
         username,
-        firstName,
-        lastName,
+        displayName,
         email,
+        group,
+      },
+      {
+        username: 'user1',
+        displayName: 'Johannes Kepler',
+        email: 'user1@myseneca.ca',
+        group: 'mycustomdomain-faculty',
+      },
+      {
+        username: 'user2',
+        displayName: 'Galileo Galilei',
+        email: 'user2@myseneca.ca',
+        group: 'mycustomdomain-students',
+      },
+      {
+        username: 'user3',
+        displayName: 'Adam Mason',
```

Comment:
cc @sfrunza13 - we better keep this seed list and the config for simplesaml and our docs in sync.

Reply:
Fixed in #312